tests/bluetooth/ctrl_sw_privacy_unit/src/main.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2020 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr/types.h>
+#include <ztest.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define CONFIG_BT_CTLR_RL_SIZE 8
```

Comment:
Shouldn't these be in the prj.conf instead?

Reply:
This is a unit test so I'm testing without the whole stack, as such I would like to avoid pulling in the whole code base